spec/factories/endpoint.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :endpoint do
+    verify_ssl OpenSSL::SSL::VERIFY_PEER
```

Comment:
sorry, I guess this should be dropped because it's already set via `default_value_for` in the model. And setting it here could lead to false positives?!